src/test/java/org/folio/innreach/domain/listener/KafkaCirculationEventListenerApiTest.java

```diff
@@ -137,6 +146,54 @@ void shouldReturnCheckedInItem() {
     verify(innReachExternalService).postInnReachApi(any(), any());
   }
 
+  @Test
+  @Sql(scripts = {
+    "classpath:db/central-server/pre-populate-central-server.sql",
+    "classpath:db/inn-reach-transaction/pre-populate-inn-reach-transaction.sql",
+  })
+  void shouldRenewalLoanToUpdateTransaction() {
+    UUID folioLoanId = UUID.fromString("fd5109c7-8934-4294-9504-c1a4a4f07c96");
+    String date = "01.07.2016";
+    SimpleDateFormat format = new SimpleDateFormat("dd.MM.yyyy");
+    Date dueDate = null;
+    try {
+      dueDate = format.parse(date);
+    } catch (Exception e) {
+      e.printStackTrace();
+    }
```

Comment:
it doesn't seem to be the best way to create dates, please optimize this.

Reply:
Done